### PR TITLE
initial commit. add writingDate function

### PR DESCRIPTION
**Pull request by night-hawks-wake**
2차 시도..!

### CODE REVIEW

**Comment by Bbarabarabam** [comment]
ok

**Review by kkerast — COMMENTED** [review]
preview ok!

**Review by Bbarabarabam — COMMENTED** [review]
ok

File: app.py [hunk 1]
```diff
@@ -1,15 +1,43 @@
...
+        weekStartsAt = weekStartsAt - timedelta(days = day)
+
+    for i in range(0, 7):
+        thisDate = weekStartsAt + timedelta(days = i)
+        thisDateStr = dateToStr(thisDate, "%Y-%m-%d")
+        weekDate.append(thisDateStr)
+    
+    return weekDate
+
 app = Flask(__name__)
```
Comment (Bbarabarabam) [inline app.py:22]: hi

File: app.py [hunk 2]
```diff
@@ -1,15 +1,43 @@
 from flask import Flask, render_template, request, jsonify
+from datetime import datetime, timedelta
+
+def dateToStr (date, pattern) : # date를 string으로 바꿔주되, pattern대로 바꿔주는 함수
```
Comment (kkerast) [inline app.py:4]: ok

**Review by Bbarabarabam — COMMENTED** [review]
ok22

**Review by kkerast — COMMENTED** [review]
ok!!

File: templates/login.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Login</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<style>
+		body {
+			font-family: Arial, sans-serif;
+			background-color: #f2f2f2;
```
Comment (Bbarabarabam) [inline templates/login.html:10]: hhi

File: templates/sign-up.html [hunk 4]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Sign Up</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<style>
+		body {
+			font-family: Arial, sans-serif;
```
Comment (Bbarabarabam) [inline templates/sign-up.html:9]: hi

File: test.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+now = datetime.today()
+day = now.weekday()
+startDate = datetime.strptime(now.strftime("%Y-%m-%d"), "%Y-%m-%d")
+weekDate = []
+if (day != 0):
+    startDate = startDate - timedelta(days=day)
+
+for i in range(0, 7):
+    date = startDate + timedelta(days=i)
+    weekDate.append(date)
```
Comment (Bbarabarabam) [inline test.py:12]: hi

**Review by Bbarabarabam — COMMENTED** [review]
k

**Pull request closed without merging**